### ISSUE TITLE
Where can I find documentation on what the colored letters mean in the new pop up window?

### ISSUE THREAD

**Issue by Rofer11607** [neoclide/coc.nvim#4186]
![image](https://user-images.githubusercontent.com/55964540/189716174-bc4ada6f-23c9-480d-91e5-15511643bb3c.png)

**Comment by chemzqm** [comment]
`:h coc-config-suggest-completionItemKindLabels`

**Comment by Rofer11607** [comment]
"suggest.completionItemKindLabels"			*coc-config-suggest-completionItemKindLabels*

	Set custom labels to completion items' kinds.

	Scope: `application`, default: `{}`

I am more looking for what does the red "k" mean or the blue "v"

**Comment by chemzqm** [comment]
https://github.com/neoclide/coc.nvim/blob/17a3e25ed37b0fa9de22e2cfe083d973cbec9c02/doc/coc-config.txt#L1104

**Comment by Rofer11607** [comment]
Clicking that link returns 404

![image](https://user-images.githubusercontent.com/55964540/189924615-089231d2-fd39-44e0-b46f-ea00f496bc88.png)

**Comment by Rofer11607** [comment]
I did :h coc-config, maybe I'm being dumb can you dumb it down for me lol?

**Comment by chemzqm** [comment]
```

"suggest.completionItemKindLabels"			*coc-config-suggest-completionItemKindLabels*

	Set custom labels to completion items' kinds.
	Default value: >
	{
	  "text": "v",
	  "method": "f",
	  "function": "f",
	  "constructor": "f",
	  "field": "m",
	  "variable": "v",
	  "class": "C",
	  "interface": "I",
	  "module": "M",
	  "property": "m",
	  "unit": "U",
	  "value": "v",
	  "enum": "E",
	  "keyword": "k",
	  "snippet": "S",
	  "color": "v",
	  "file": "F",
	  "reference": "r",
	  "folder": "F",
	  "enumMember": "m",
	  "constant": "v",
	  "struct": "S",
	  "event": "E",
	  "operator": "O",
	  "typeParameter": "T",
	  "default": ""
	}
<

```
